### PR TITLE
Added docker build to CI

### PR DESCRIPTION
**Pull request by JoeAmedeo**
This PR is intended to add an additional step to the CI action, as a means of detecting docker build errors prior to PR completion.

#### ✔ Checklist:

- [x] This PR has been added to [CHANGELOG.md](https://github.com/dotenv-linter/dotenv-linter/blob/master/CHANGELOG.md) (at the top of the list);
- [x] Tests for the changes have been added (for bug fixes / features);
- [x] Docs have been added / updated (for bug fixes / features).

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -92,6 +92,14 @@ jobs:
   #       token: ${{ secrets.GITHUB_TOKEN }}
   #       args: --all-features -- -D warnings
 
+  docker:
+    name: Build / Docker
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Build
+        run: sudo docker build --target builder .
```
Comment (mgrachev) [inline .github/workflows/ci.yml:101]: We don't use the `sudo` command in the [release-docker](https://github.com/dotenv-linter/dotenv-linter/blob/master/.github/workflows/release.yml#L59) job. Is it necessary here?

**Review by mgrachev — COMMENTED** [review]
@JoeAmedeo 👋 Thank you for your contribution! 👍

I have left a comment. Please look at it 👀

File: .github/workflows/ci.yml (hunk 1 above)
Comment (JoeAmedeo) [reply .github/workflows/ci.yml]: good point, It is not needed. `sudo` has been removed from the command.

**Review by mgrachev — APPROVED** [review]

**Comment by mgrachev** [comment]
@JoeAmedeo Thank you! 💪

🙏 If it’s not difficult for you, please support the project - click on the star on GitHub ⭐️

**Pull request merged**